### DISCUSSION TITLE
SCC Not converged while calculating band structure

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1463] [Q&A] [answered]
Hello everyone, 

I have tried to calculate the band structure for a high atomic system using DFTB+. As referred to the recipe and manual, I found that we have to set MaxSCCIterations = 1 in the Hamiltonian section. It calculates the band_tot.dat file but it displays, SCC not converged.

Would it effect the calculation? since for the band_structure calculation, our concern is not to converge the scc so we give the value 1 for its iteration.

I am confused if SCC NOT converged will be a problem in my calcuation.

Please respond me!!

**Comment by aradi** [comment, marked as answer]
The message is correct, but maybe not the best, indeed. If you do band structure calculation, you want to avoid SCC convergence, as you want to do only one step with the charges you converged in previous a calculation with proper k-point sampling. This is why you set  `MaxSccIterations = 1`. The message only means, that your density is not converged with respect of the k-point set you use for the band structure calculation. (Which is OK, because the k-point sampling would provide a very poor BZ-average...)
